src/Sentry.Protocol/Context/Contexts.cs

```diff
@@ -59,13 +59,8 @@ internal Contexts Clone()
         /// <summary>
         /// Copies the items of the context while cloning the known types.
         /// </summary>
-        internal void CopyTo(Contexts? to)
+        internal void CopyTo(Contexts to)
         {
-            if (to == null)
```

Comment:
Happy to kill these null checks on `internal` method btw, where we can guarantee we only call from C# 8+

Reply:
There's a bit of danger though as some internal methods are accessed from tests. I didn't want to accidentally break any of them so refrained from doing it.